src/pydata_sphinx_theme/assets/styles/variables/_color.scss

```diff
@@ -284,6 +284,7 @@ $pst-semantic-colors: (
     --pst-color-heading: var(--pst-color-text-base);
     --pst-color-link: var(--pst-color-primary);
     --pst-color-link-hover: var(--pst-color-secondary);
+    --pst-color-link-higher-contrast: var(--pst-color-inline-code-links);
```

Comment:
Would it not be best to rename `--pst-color-inline-code-links` to `--pst-color-link-higher-contrast` vs creating another variable? 
I used `--pst-color-inline-code-links` originally since I did not realise we'd need to use it in multiple places.

Reply:
Yeah, I'm fine with that. My original thinking was that inline code links might be worth keeping semantically distinct. But that means I should have written the variable aliasing in the other direction:

```scss
--pst-color-inline-code-links: var(--pst-color-link-higher-contrast);
```

But I'm fine with collapsing it all.

Reply:
Yeah that other aliasing direction makes more sense (I did not think about it originally). But I think either if fine, thanks.